### PR TITLE
[Merged by Bors] - [CU-3409vc7] fix need for 'eager: true' in pablo model

### PR DESCRIPTION
**Pull request by andresmechali**
## Issue
https://app.clickup.com/t/3409vc7

## Description
Besides from the fix this applies `prettier`, but the only addition is `relations: ["poolAssets"],`

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-3409vc7 Fix need of adding eager: true  to Pablo model every time types are autogenerated](https://app.clickup.com/t/3409vc7)

File: subsquid/src/dbHelper.ts [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+  poolId: bigint
 ): Promise<PabloPool | undefined> {
-    return await store.get<PabloPool>(PabloPool, {
-        where: {poolId},
-        order: {calculatedTimestamp: 'DESC'},
-    });
+  return await store.get<PabloPool>(PabloPool, {
+    where: { poolId },
+    order: { calculatedTimestamp: "DESC" },
+    relations: ["poolAssets"],
```
Comment (vimukthi-git) [inline subsquid/src/dbHelper.ts:21]: I'm not sure if this is the right solution, as this requires any function that calls the db to explicitly state the which ones to load eagerly.

**Review by vimukthi-git — APPROVED** [review]
LGTM

**Review by Dom-Roth — APPROVED** [review]
LGTM

**Review by easteregg — APPROVED** [review]

**Review by KaiserKarel — APPROVED** [review]

**Pull request closed without merging**